File: Python/ProgLabHW5/RestaurantNames.py
from enum import Enum, unique


@unique
class RestaurantName(Enum):
    Delice = 1
    EuroHotel = 2
    Morio = 3

    def __gt__(self, other):
        return self.name.lower() > other.name.lower()

    def __str__(self):
        return self.name

File: C#/homework4/EventApi/RestaurantNames.cs
using NpgsqlTypes;

namespace EventApi
{
    /// <summary>
    /// Restaurant name where event is
    /// </summary>
    /// <example>0</example>
    public enum RestaurantNames
    {
        [PgName("Delice")]
        Delice,
        [PgName("EuroHotel")]
        EuroHotel,
        [PgName("Morio")]
        Morio
    }
}